### PR TITLE
Support matching untagged packet

### PR DESCRIPTION
**Pull request by gran-vmv**
Signed-off-by: gran <gran@vmware.com>

### CODE REVIEW

File: openflow13/match.go [hunk 1]
```diff
@@ -805,6 +805,20 @@ func NewVlanIdField(vlanId uint16, vlanMask *uint16) *MatchField {
 	return f
 }
 
+// Return a MatchField for non-vlan matching
```
[the remark this answers is not in the document]
Comment (gran-vmv) [reply openflow13/match.go:808]: As this line said, it is to match packet without VLAN tag.
Existing `NewVlanIdField()` forcibly adds `OFPVID_PRESENT` to `vlanIdField.VlanId` https://github.com/antrea-io/libOpenflow/blob/main/openflow13/match.go#L794
Thus we cannot match non-VLAN packets without adding new functions.

[the remark this answers is not in the document]
Comment (gran-vmv) [reply openflow13/match.go:808]: This new function is to support the second case of `OXM_OF_VLAN_VID`.
```
              Fully wildcarded
                     Matches any packet, that is, one without an 802.1Q
                     header or with an 802.1Q header with any TCI value.

              Value 0x0000 (OFPVID_NONE), mask 0xffff (or no mask)
                     Matches only packets without an 802.1Q header.

              Value 0x1000, mask 0x1000
                     Matches any packet with an 802.1Q header,
                     regardless of VLAN ID.

              Value 0x1009, mask 0xffff (or no mask)
                     Match only packets with an 802.1Q header with VLAN
                     ID 9.

              Value 0x1001, mask 0x1001
                     Matches only packets that have an 802.1Q header
                     with an odd-numbered VLAN ID. (This is just an
                     example; one can match on any desired VLAN ID bit
                     pattern.)
```

[the remark this answers is not in the document]
Comment (wenyingd) [reply openflow13/match.go:808]: So you mean if we have installed one flow entry matching VLAN, the packets without VLAN tag is not able to hit by other flow entries even if we don't add vlan match?

**Comment by wenyingd** [comment]
This PR is not necessary, since the VlanId field is public. If the only change is to update its value, we could let caller to deside the final values.

**Pull request closed without merging**